esx_service/utils/auth_api.py

```diff
@@ -527,8 +529,7 @@ def _tenant_vm_add(name, vm_list):
         return error_info
 
     logging.debug("_tenant_vm_add: vms=%s", vms)
-    vms_uuid_list = [(vm_id) for (vm_id, vm_name) in vms]
-    error_msg = tenant.add_vms(auth_mgr.conn, vms_uuid_list)
+    error_msg = tenant.add_vms(auth_mgr.conn, vms)
```

Comment:
add_vms is expecting only a list of VM uuids vs. a tuple of (uuid, name), is it ok to pass vms here?